xacro/torso.xacro

```diff
@@ -119,12 +119,26 @@
                     rpy="0 0 0" xyz="0 0 0"
                     mass="0.061"
                     len_x="0.0175" len_y="0.091" len_z="0.028" />
-	  </link>
-	  <joint name="camera_fix" type="fixed">
-	    <origin rpy="0 0 0" xyz="0.045 0 0"/>
-	    <parent link="camera_support"/>
-	    <child link="camera"/>
-	  </joint>
+    </link>
```

Comment:
Can you keep the same indentation for the diff? (``Files changed`` tab of the PR on GitHub)